app/locations/middleware.js

```diff
@@ -9,7 +9,6 @@ function checkLocationsLength(req, res, next) {
   if (req.userLocations.length === 1) {
     return res.redirect(`${req.baseUrl}/${req.userLocations[0].id}`)
   }
-
```

Comment:
Rogue change crept in?

Reply:
Yes -- I had removed one blank line. Now duly restored.